documentation/src/docs/reader-group-design.md

```diff
@@ -11,83 +11,85 @@ You may obtain a copy of the License at
 
 _To be updated to reflect recent implementation details (05/07/2017)_
 
-## Motivation 
-A set of Readers can be grouped together in order that the set of Events in a Stream can be read in parallel.  The grouping of Readers is called a Readers Group.  Pravega guarantees that each Event in the Stream is read by exactly one Reader in the Readers Group. 
+## Motivation
+A set of Readers can be grouped together in order that the set of Events in a Stream can be read in parallel. This grouping of Readers is called a Reader Group. Pravega guarantees that each Event in the Stream is read by exactly one Reader in the Reader Group.
 
-Each Reader in a ReaderGroup is assigned zero or more Segments.
-The Reader assigned to a Segment is the only Reader within the ReaderGroup that reads Events from that Segment.  This is the fundamental mechanism by which Pravega makes ordering guarantees of Event delivery to a Reader – a Reader will receive Events in the order they were published into a Segment.
+Each Reader in a Reader Group is assigned zero or more Segments.
+The Reader assigned to a Segment is the only Reader within the Reader Group that reads Events from that Segment. This is the fundamental mechanism by which Pravega makes ordering guarantees of Event delivery to a Reader – a Reader will receive Events in the order they were published into a Segment.
 There are several challenges associated with this mechanism:
 
-- How to maintain the mapping of which Reader within a ReaderGroup is assigned which Segment
-- How to manage the above mapping when Segments split and merge
-- How to manage the above mapping when Readers are added to the ReaderGroup 
-- How to manage the above mapping when Readers leave the ReaderGroup either by an explicit operation or the Reader becoming unavailable due to network outage or the Reader process failing
+ -  How to maintain the mapping of which Reader within a Reader Group is assigned which Segment?
+ -  How to manage the above mapping when Segments split and merge?
+ -  How to manage the above mapping when Readers are added to the Reader Group?
+ -  How to manage the above mapping when Readers leave the Reader Group either by an explicit operation or the Reader becoming unavailable due to network outage or the Reader process failing?
 
-To solve these problems we can use [StateSynchronizer](state-synchronizer-design.md) to enable readers to coordinate among themselves.
+To address these challenges, we can use [State Synchronizer](state-synchronizer-design.md) to enable coordination among Readers.
```

Comment:
Here `StateSynchronizer` is a single word.

Reply:
Hi @RaulGracia,
As we discussed, we need to decide on the naming conventions... (Reader Group, Reader/Readers).
Hope in this docs the terms are edited correctly?

Reply:
@Aparnarr

Reply:
@fpj Please include @RaulGracia  as another reviewer